README.md

```diff
@@ -6,37 +6,34 @@
 <br>
 <p align="center">
   <a href="https://www.ruby-lang.org/en/">
-    <img src="https://img.shields.io/badge/Ruby-v2.7.1-green.svg" alt="ruby version">
+    <img src="https://img.shields.io/badge/Ruby-v2.7.2-green.svg" alt="ruby version">
   </a>
   <a href="http://rubyonrails.org/">
     <img src="https://img.shields.io/badge/Rails-v6.0.3-brightgreen.svg" alt="rails version">
   </a>
-  <a href="https://travis-ci.com/thepracticaldev/dev.to">
-    <img src="https://travis-ci.com/thepracticaldev/dev.to.svg?branch=master" alt="Travis Status for thepracticaldev/dev.to">
+  <a href="https://travis-ci.com/forem/forem">
+    <img src="https://travis-ci.com/forem/forem.svg?branch=master" alt="Travis Status for forem/forem">
   </a>
-  <a href="https://codeclimate.com/github/thepracticaldev/dev.to/maintainability">
+  <a href="https://codeclimate.com/github/forem/forem/maintainability">
     <img src="https://api.codeclimate.com/v1/badges/ce45bf63293073364bcb/maintainability" alt="Code Climate maintainability">
   </a>
-  <a href="https://codeclimate.com/github/thepracticaldev/dev.to/test_coverage">
```

Comment:
I completely removed this, since CC said this isn't even configured for our repo.